hugo/content/info/corrections.md

```diff
@@ -32,21 +32,32 @@ Please take note of the following regarding errata and revisions:
 
 ### Correcting Metadata
 
-For requests to change paper *metadata* (that do not require any PDF changes), please file [a Github issue](https://github.com/acl-org/acl-anthology/issues/new).
+For requests to change paper *metadata* (that do not require any PDF changes), please file [a Github issue](https://github.com/acl-org/acl-anthology/issues/new).[^1]
 **Be sure to indicate the Anthology ID of the paper** (e.g., `P19-1017`).
+Metadata changes are generally accepted if they comport with the PDF, which we take as authoritative.
+
+[^1]: We prefer to work through Github issues since it simplifies our workflow.
```

Comment:
The sensitive-issue issue seems somewhat important to me. What would you think about structuring these instructions as:

A request to change paper *metadata* (that does not require any PDF changes) can be submitted in several ways. 

1. (Preferred) Please file [a Github issue](https://github.com/acl-org/acl-anthology/issues/new). **Be sure to indicate the Anthology ID of the paper** (e.g., `P19-1017`).

2. If your issue is sensitive, you can alternately contact us via email at anthology@aclweb.org. Again, please be sure to include the Anthology ID of the paper in your email.

3. If you would like to expedite the process, ...

The Anthology team will attend to the correction as we find time. Metadata changes are generally accepted if they are consistent with the PDF, which we take as authoritative.

Reply:
Thanks for the careful read, this is great feedback.